plenum/server/monitor.py

```diff
@@ -52,9 +53,11 @@ def is_ordered(self):
         def is_ordered_by_all(self):
             return all(self.ordered)
 
-    def __init__(self, instance_count):
+    def __init__(self, instance_count, config=None):
         self.instance_count = instance_count
         self._requests = {}
+        self._messaged_reqs = deque()
```

Comment:
Please add a comment what `_messaged_reqs` is or maybe rename it to something like `_unordered_req`?